packages/block-library/src/paragraph/edit.js

```diff
@@ -98,32 +130,13 @@ function ParagraphBlock( {
 					}
 				/>
 			</BlockControls>
-			{ isDropCapFeatureEnabled && (
-				<InspectorControls group="typography">
-					<ToolsPanelItem
-						hasValue={ () => !! dropCap }
-						label={ __( 'Drop cap' ) }
-						onDeselect={ () =>
-							setAttributes( { dropCap: undefined } )
-						}
-						resetAllFilter={ () => ( { dropCap: undefined } ) }
-						panelId={ clientId }
-					>
-						<ToggleControl
-							__nextHasNoMarginBottom
-							label={ __( 'Drop cap' ) }
-							checked={ !! dropCap }
-							onChange={ () =>
-								setAttributes( { dropCap: ! dropCap } )
-							}
-							help={ helpText }
-							disabled={
-								hasDropCapDisabled( align ) ? true : false
-							}
-						/>
-					</ToolsPanelItem>
-				</InspectorControls>
-			) }
+			<InspectorControls group="typography">
+				<DropCapControl
```

Comment:
So you think the useSetting call is more expensive than rendering `InspectorControls`?

Reply:
oh the useSetting is only done for the selected block now. Makes sense.

Reply:
Yes, because (1) drop cap is supported by default and (2) individual InspectorControls instance no longer make select calls but read from context instead, so they're much cheaper.